sql/moz-fx-data-shared-prod/monitoring_derived/airflow_task_instance_v1/schema.yaml

```diff
@@ -19,10 +19,6 @@ fields:
   mode: NULLABLE
   name: try_number
   type: INTEGER
-- description:
```

Comment:
Is this field not in the table?

Reply:
It does, I just moved it below to match query and schema.